README.md

```diff
@@ -59,3 +60,6 @@ This is where controls code and driver code lives. Abstracts the vehicle from th
 
 ## tauv_commander
 This is the package for the operator interface. TODO.
+
+## uuv-simulator
+This is the package that handles the simulator.
```

Comment:
In addition to this, you should create a README.md inside the thruster_manager folder in tauv_common, where you describe what the publish/subscribed topics are, where configuration files are stored, etc.